typings/index.d.ts

```diff
@@ -1455,8 +1457,10 @@ declare module 'discord.js' {
 		token: string;
 		delete(reason?: string): Promise<void>;
 		edit(options: WebhookEditData): Promise<Webhook>;
-		send(content?: StringResolvable, options?: WebhookMessageOptions | MessageAdditions): Promise<Message | Message[]>;
-		send(options?: WebhookMessageOptions | MessageAdditions | APIMessage): Promise<Message | Message[]>;
+		send(content?: StringResolvable, options?: WebhookMessageOptions & { split: true }): Promise<Message[]>;
+		send(content?: StringResolvable, options?: WebhookMessageOptions & { split?: false } | MessageAdditions): Promise<Message>;
+		send(options?: WebhookMessageOptions & { split: true }): Promise<Message[]>;
+		send(options?: WebhookMessageOptions & { split?: false } | MessageAdditions | APIMessage): Promise<Message>;
```

Comment:
I think you are missing SplitOptions in these.

Reply:
Fixed, thank you!